### DISCUSSION TITLE
Not all environment protection rules are available

### DISCUSSION THREAD

**Discussion by smrafi** [community/community#32229] [Actions]
Hi, I'm trying to setup deployment based on environment as it was explained in https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment . However when I was trying to create environment protection rules, I don't find Required reviewers and wait timer options available for me. I only see branch protection rules and secrets. Do I miss something or should I enable some settings to make it available?

![Screenshot from 2022-09-07 16-47-53](https://user-images.githubusercontent.com/11792249/188908594-1d470e6b-adbc-45d2-9f71-52a17cab5d28.png)

I'm an administrator for my organization and we use Github Teams. Your help and support is really appreciated.
Thanks in advance!

**Comment by BredoGen** [comment]
It is very unclear but this simple feature is restricted only to Enterprise plan

**Reply by ikondrasovas** [reply]
If this is the case, then how subscribers from "Team" and other plans can realize the "[build once, deploy many times](https://learn.microsoft.com/en-us/dotnet/architecture/devops-for-aspnet-developers/actions-build)" principle mentioned: 
![image](https://user-images.githubusercontent.com/6977172/191102153-323e2305-7dd3-4213-bd46-6ad60019da4f.png) 

Or [deploy the same binary in each environment](https://learn.microsoft.com/en-us/dotnet/architecture/devops-for-aspnet-developers/actions-deploy):

![image](https://user-images.githubusercontent.com/6977172/191103321-e15136c2-4ec8-4340-9544-9602abdc8f59.png)


Any ideas on how to do that in an automated fashion?


Thanks!

**Reply by smeijer** [reply]
Yeah it sucks, seems like the only option is to make your repo public, or to upgrade to enterprise:

<img width="1323" alt="image" src="https://user-images.githubusercontent.com/1196524/191284068-c58b38f5-da20-4c55-b40c-c13d193cab3c.png">

**Reply by James-Firth** [reply]
Annoying as Azure DevOps Pipelines have this feature down pat (I believe at a non-enterprise tier but I could be wrong)

**Reply by AzySir** [reply]
Anyone say Gitlab...
